### ISSUE TITLE
backend: need to setup mongoDB

### ISSUE THREAD

**Issue by himanshujaidka** [DevMan-open-source/dailymint#4]
Need to set up MongoDB and make a connection to the server.

**Comment by varisha-025** [comment]
Hello @himanshujaidka. I am a participant in GSSoC 2022 and I would like to work on this issue. Can you explain the next steps on this?

**Comment by himanshujaidka** [comment]
Yes but wait for the official start date till then read some documentation regarding mongoDB

**Comment by Gurdeep475** [comment]
I would like to work on this issue under GSSOC`22.

**Comment by varisha-025** [comment]
Hey @himanshujaidka I would like to work on this issue under GSSOC , please assign it to me.

**Comment by haidyelnahass** [comment]
I would like to work on this issue under GSSOC

**Comment by lokita282** [comment]
I am a gssoc '22 participant and would like to work on this issue

**Comment by RaviAnand111** [comment]
Hey @himanshujaidka , as @varisha-025 , removed the assignment I would like to work on this issue, but I have this doubt, that there is no server to connect to.

**Comment by Gurdeep475** [comment]
@himanshujaidka, I would like to work on this issue, please assign it to me, since I was next in the line after @varisha-025.
